docs/io/fits/appendix/faq.rst

```diff
@@ -52,19 +52,14 @@ bases is non-trivial.  The second is that there are many features of Astropy
 greatly.  Since PyFITS is already integrated into Astropy, it makes more sense
 to continue development there rather than make Astropy a dependency of PyFITS.
 
-PyFITS' current primary developer and active maintainer is `Erik Bray`_, though
-patch submissions are welcome from anyone.  PyFITS is now primarily developed
-in a Git repository for ease of merging to and from Astropy.  Patches and issue
-reports can be posted to the `GitHub project`_ for PyFITS, or for Astropy.
+PyFITS' past primary developer and active maintainer was `Erik Bray`_.  There
+is a `GitHub project`_ for PyFITS but PyFITS is not actively developed anymore
+so patches and issue reports should be posted on the Astropy issue tracker.
 There is also a legacy `Trac site`_ with some older issue reports still open,
 but new issues should be submitted via GitHub if possible.  An `SVN mirror`_ of
```

Comment:
Remove the SVN mirror. It is locked, so there is no point to mention it. On the other hand, you can leave Trac link in because there might be some issues there that don't exist on GitHub (both PyFITS's and Astropy's).